### PR TITLE
[OGM] Adding part 3 of "Hibernate OGM 101" series: Running on WildFly

### PR DESCRIPTION
**Pull request by gunnarmorling**
@DavideD, here is the example for the blog post :)

### CODE REVIEW

**Comment by DavideD** [comment]
Looking at ti

**Comment by DavideD** [comment]
PersonsIT test fails for me. I'm trying to find the cause

**Comment by DavideD** [comment]
If the user is not in the Berlin time-zone the test will fail. I would change it like in this commit:
https://github.com/hibernate/hibernate-demos/commit/faeaf23ae38870d6da34de2d8c4821e1dd593766

File: hibernate-ogm/nosql-with-hibernate-ogm-101/hibernate-ogm-demo-nosql-with-hibernate-ogm-101-part-3/src/main/java/org/hibernate/ogm/demos/ogm101/part3/repo/HikeRepository.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+import org.hibernate.ogm.demos.ogm101.part3.model.Hike;
+
+/**
+ * @author Gunnar Morling
+ */
+@ApplicationScoped
+public class HikeRepository {
```
Comment (DavideD) [inline hibernate-ogm/nosql-with-hibernate-ogm-101/hibernate-ogm-demo-nosql-with-hibernate-ogm-101-part-3/src/main/java/org/hibernate/ogm/demos/ogm101/part3/repo/HikeRepository.java]: It is pretty obvious but I would add a comment to explain what this class does

File: hibernate-ogm/nosql-with-hibernate-ogm-101/hibernate-ogm-demo-nosql-with-hibernate-ogm-101-part-3/src/main/java/org/hibernate/ogm/demos/ogm101/part3/repo/PersonRepository.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import javax.persistence.PersistenceContext;
+
+import org.hibernate.ogm.demos.ogm101.part3.model.Hike;
+import org.hibernate.ogm.demos.ogm101.part3.model.Person;
+
+/**
+ * @author Gunnar Morling
+ */
+@ApplicationScoped
+public class PersonRepository {
```
Comment (DavideD) [inline hibernate-ogm/nosql-with-hibernate-ogm-101/hibernate-ogm-demo-nosql-with-hibernate-ogm-101-part-3/src/main/java/org/hibernate/ogm/demos/ogm101/part3/repo/PersonRepository.java]: Same here: It is pretty obvious but I would add a comment to explain what this class does

**Comment by DavideD** [comment]
I''ve added an additional commit with some javadoc on my branch: ff23f2e880dd885b897d05416c8dc3477dec2d71

**Comment by gunnarmorling** [comment]
Cool, thanks! Then go for it and push it :)

**Comment by DavideD** [comment]
I've squashed the two commits I mention before and merged everything

Thanks.

**Comment by DavideD** [comment]
Merged https://github.com/hibernate/hibernate-demos/pull/5

**Pull request closed without merging**